src/resources/MachineLearning/RGAConfiguration/RGAConfiguration.ts

```diff
@@ -0,0 +1,12 @@
+import API from '../../../APICore.js';
+import Resource from '../../Resource.js';
+import {RGADocumentGroupPreview, RGAPreviewParams} from './RGAConfigurationInterface.js';
+
+export default class RGAConfiguration extends Resource {
```

Comment:
Should we use the full name everywhere? I feel using acronyms like RGA adds needlessly to the Coveo learning curve

Reply:
I am neutral about this, its true that it will take sometime to make the connection between RGA and Relevance Generative Answering. But the p-c is used for more technical external users. The doc they will be referring to will already establish the connection between the short-hand names and the long name: https://platformdev.cloud.coveo.com/docs?urls.primaryName=Machine%20Learning%20Configuration#/Relevance%20Generative%20Answering%20Configuration/rest_organizations_paramId_machinelearning_configuration_rga_preview_post

Reply:
Personally, I prefer the full name instead of acronym, it's clearer. 

It's way more confusing for someone new to comme across RGA or IAPR, than RelevanceGenerativeAnswering and IntentAwareProductRanking

Reply:
I updated all interfaces to use the full name! Thanks